### ISSUE TITLE
opts

### ISSUE THREAD

**Issue by hyimarm** [Rowenazhang/SVTformer#1]
Hello, there is a missing superparameter in the code you provided. Setting the superparameter of the multi view fusion convolution layer to 1 or 3 by myself does not reproduce the effect in your paper. Can you provide me with the value of this superparameter

**Comment by Rowenazhang** [comment]
Sorry for the confusion. In our experiments, it was set to **7**.

I have added it to `opt.py`:

```python
self.parser.add_argument('--mvf_kernel', default=7, type=int)

**Comment by sixsixsix2025** [comment]
Hi, thanks for releasing the code for SVTFormer.
I noticed that in model/svtformer.py, there is a module defined as:
self.conv = nn.Sequential(
    nn.BatchNorm2d(4, momentum=0.1),
    nn.Conv2d(4, 1, kernel_size=opt.mvf_kernel, stride=1,
              padding=int(opt.mvf_kernel // 2), bias=False),
    nn.ReLU(inplace=True),
)
and --mvf_kernel is also defined in common/opt.py.
However, I could not find any usage of self.conv(...) in the forward() function, and I also did not see mvf_kernel mentioned in the paper. The actual multi-view interaction seems to be implemented through the View Transformer blocks instead.
Could you please confirm whether self.conv / mvf_kernel is an unused leftover from an earlier multi-view fusion ablation, or whether it is intended to be used somewhere in the official model?
Thank you!
